sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_AnalyzeBatchActions.md

```diff
@@ -39,7 +52,13 @@ To run analyze operation in multiple documents, call `StartAnalyzeOperationBatch
 
     AnalyzeOperation operation = client.StartAnalyzeOperationBatch(batchDocuments, operationOptions);
 
-    await operation.WaitForCompletionAsync();
+    TimeSpan pollingInterval = new TimeSpan(1000);
+
+    while (!operation.HasCompleted)
+    {
+        Thread.Sleep(pollingInterval);
+        operation.UpdateStatus();
+    }
```

Comment:
for the end of the file where there are links, consider adding also Sample_AnalyzeOperationBatchConvenience.cs and the async version. something like:
Synchronous AnalyzeBathActions 
Asynchronous AnalyzeBathActions 
Synchronous AnalyzeBathActionsConvenience
Asynchronous AnalyzeBathActionsConvenience